pkg/controller/machine/controller.go

```diff
@@ -365,9 +365,12 @@ func (r *ReconcileMachine) drainNode(machine *machinev1.Machine) error {
 	}
 
 	if nodeIsUnreachable(node) {
-		klog.Infof("%q: Node %q is unreachable, draining will wait %d seconds after pod is signalled for deletion and skip after it",
-			machine.Name, node.Name, skipWaitForDeleteTimeoutSeconds)
+		klog.Infof("%q: Node %q is unreachable, draining will wait ignore gracePeriod. PDBs are still honored.",
```

Comment:
Extra word in here I think?
```suggestion
		klog.Infof("%q: Node %q is unreachable, draining will ignore gracePeriod. PDBs are still honored.",
```